File: src/data/Model/Cms360/AcademicCalendar/AcademicCalendarMaster.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
//sadfsd
namespace Cms360.Data.Model
{

    [Table("AcademicCalendarMaster", Schema = "AcademicCalendar")]

    public class AcademicCalendarMaster
    {
        [Key]
        public Guid AcademicCalendarMasterId { get; set; }
        public Guid SessionId { get; set; }
        public Guid SubCityId { get; set; }
        public Guid ClassId { get; set; }
        public Guid? BoardId { get; set; }
        public string FullName { get; set; }
        public DateTime FromDate { get; set; }
        public DateTime ToDate { get; set; }
        public int Weeks { get; set; }
        public int StatusId { get; set; }
        public int YearlyWeekNo { get; set; }
        public Boolean? IsApproved { get; set; }
    }
    
    public class AcademicCalendarMasterCity
    {
        [Key]
        public Guid AcademicCalendarMasterId { get; set; }
        public Guid SessionId { get; set; }
        public Guid SubCityId { get; set; }
        public Guid ClassId { get; set; }
        public Guid? BoardId { get; set; }
        public string FullName { get; set; }
        public DateTime FromDate { get; set; }
        public DateTime ToDate { get; set; }
        public int Weeks { get; set; }
        public int StatusId { get; set; }
        public int YearlyWeekNo { get; set; }
        public Boolean? IsApproved { get; set; }
                public string SubCity { get; set; }

    }
}

File: src/ui/app/root/application/models/academiccalendar/academiccalendarmaster.ts
export interface IAcademicCalendarMaster {
    academicCalendarMasterId: string;
    sessionId: string;
    subCityId: string;
    classId: string;
    boardId?: string;
    fullName: string;
    fromDate: Date;
    toDate: Date;
    weeks: number;
    statusId: number;
    isApproved: Boolean;
}

export interface IAcademicCalendarMasterCity {
    academicCalendarMasterId: string;
    sessionId: string;
    subCityId: string;
    classId: string;
    boardId?: string;
    fullName: string;
    fromDate: Date;
    toDate: Date;
    weeks: number;
    statusId: number;
    isApproved: Boolean;
    subCity: string;
}